### PR TITLE
Add videos link in header and footer

### PR DESCRIPTION
**Pull request by JamesScript7**
Hello @hzoo,

This was the video link we were talking about at the hacktoberfest event. Fairly simple contribution, tested out responsiveness, and works as expected. Hopefully this makes the content page more discoverable and easier to get to?

### CODE REVIEW

File: website/core/Footer.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ class Footer extends React.Component {
           </div>
           <div>
             <h5>More</h5>
+            <a href={this.props.config.getVideoUrl("videos.html", this.props.language)}>Videos</a>
```
[the remark this answers is not in the document]
Comment (JamesScript7) [reply website/core/Footer.js]: @hzoo is directly underneath Community and right above User Showcase okay?

[the remark this answers is not in the document]
Comment (hzoo) [reply website/core/Footer.js]: Up to you, not sure in that case actually? Sounds fine? I guess Stack Overflow/Slack/Twitter go together.. Or maybe we should move blog to that. ahh just thinking too much now.

Let's just go with it, can always change later.

[the remark this answers is not in the document]
Comment (JamesScript7) [reply website/core/Footer.js]: hahaha, yea, I'll leave it as the first one

**Comment by JamesScript7** [comment]
@hzoo okie dokes, updated! Thank you for this suggestion, will be one of many contributions to open source!

**Comment by hzoo** [comment]
Great thanks

**Pull request merged**